scania-digital-design-system/tegel
# docs(contribution): post feedback update

**Describe pull-request**  
Updating contribution docs

*Solving issue**  
Fixes: [CDEP-3255](https://tegel.atlassian.net/browse/CDEP-3255)

**How to test**  
1. Check and comment if something is off, or needs update or something is missing that might be of help for contributors.
